tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+    "\n",
+    "`% mkdir setup`\\\n",
+    "`% cd setup`\n",
+    "\n",
+    "Now, you need to load GROMACS, which is the simulation software package that you will utilise:\n",
+    "\n",
+    "`% module load gpu/gromacs/2020.1`\n",
+    "\n",
+    "Check that it was successfully loaded, by typing:\n",
+    "\n",
+    "`% gmx help commands`\n",
+    "\n",
+    "This will print basic information for every built-in GROMACS command. If you need more detailed information on a command in particular, you can type:\n",
+    "\n",
+    "`% gmx [command] -h`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Specifics to GROMACS\n",
+    "\n",
+    "GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. (http://www.gromacs.org/)\n",
+    "\n",
+    "- Coordinates files have the extension .gro and the default name is conf.gro.\n",
+    "- The topology file (default name topol.top) contains all the information about which atoms are bonded to which and what force-field parameters are applied etc.\n",
+    "- The trajectory files have the extension .xtc and .trr, the former does not contain velocity information and coordinates are held at a reduced precision, and so occupies less disk space. However you will need velocities if you want to continue a simulation.\n",
+    "- The .edr file contains the energy information from the trajectory.\n",
+    "- The .mdp file contains the information that was used to setup the actual simulation. Things related to temperature, pressure, how the electrostatics is calculated etc. Although we will not use this today, the file is provided so you can see how this trajectory was made.\n",
+    "- The .ndx file allows you to specify atoms or groups of atoms for use in analysis, restraints, etc and is optional.\n",
+    "- The .tpr file is a binary file that contains all the information needed to perform the actual run (this allows  gromacs to do lots of self-consistency checks to minimize user errors)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Molecular Dynamics Simulation of Protein\n",
+    "\n",
+    "### Protein System Setup\n",
+    "\n",
+    "In this section, we will obtain our protein coordinates and perform some routine Molecular Dynamics calculations on them.\n",
+    "\n",
+    "For this tutorial, we will use the HIV-1 protease structure (1HSG). It is a homodimer with two chains of 99 residues each. The .pdb file of the protein can be obtained from the protein data bank (https://www.rcsb.org/), but it is probably easier for now to just copy it from the shared directory to your working directory. In fact, most of the files you need can be found in this directory so if you get confused/stuck/lost etc then you can always look here to check you did things correctly.\n",
```

Comment:
It may be best here to link back to the previous notebook, say something like "as previously described in notebook 02, we will be simulating the structure of HV-1 protease...". And then mention how they could have uploaded the PDB files through sftp, but we did this for them already.

Reply:
Done